pages/styles/Discover.css

```diff
@@ -0,0 +1,5 @@
+.section {
+  border: 3px;
+  border-color: black;
+  border-radius: 4px;
+}
```

Comment:
Please only use styled components. That means not writing css files and not writing css classes.